service/plugins/perf/static/js/trend_chart.js

```diff
@@ -301,30 +379,78 @@ var drawSingleTrendChart = function(
     // Reduce number of calls if hash didn't changed
     if (hash != scope.currentHash) {
       scope.currentHash = hash;
+      scope.currentHashDate = d.startedAt
       scope.$emit('hashChanged', hash)
       // FIXME This slowdowns the chart dramatically
       scope.$digest()
     }
   }
 
+  // Returns list of y-positions of ops labels for given
+  // yScaledValues list.
+  function getOpsLabelYPosition(yScaledValues) {
```

Comment:
Is this something we could write a unit test for? I've recently made it so that all Jasmine tests in public/static/js/tests will be run in Evergreen

Reply:
Sure, I strongly agree we should use unit tests.

Reply:
Unfortunately, I couldn't use `make js-test` to test perf plugin, because it's separate project. I think, the UI code structure should be updated to follow best practices, however it will be a major redesign.